### PR TITLE
fix: create serivce with nonexistent variables

### PR DESCRIPTION
**Pull request by rainfd**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**
Create a service that references a variable that does not exist, request return failed, but the service is created.

**Solution:**
Validate variables before handling request.

**Related  #Issue:**
https://github.com/seal-io/walrus/issues/1003

### CODE REVIEW

**Comment by alexcodelf** [comment]
`make lint` before commit

File: pkg/apis/service/basic_view.go [hunk 1]
```diff
@@ -6,6 +6,10 @@ import (
 	"fmt"
 	"net/http"
 
+	"github.com/seal-io/walrus/pkg/dao/types/property"
+	"github.com/seal-io/walrus/pkg/deployer/terraform"
+	"github.com/seal-io/walrus/utils/json"
+
 	"k8s.io/apimachinery/pkg/util/sets"
 
 	"github.com/seal-io/walrus/pkg/apis/runtime"
```
Comment (thxCode) [inline pkg/apis/service/basic_view.go:15]: goimport

File: pkg/apis/service/basic_view.go [hunk 2]
```diff
@@ -324,3 +340,25 @@ func validateRevisionsStatus(ctx context.Context, mc model.ClientSet, ids ...obj
...
+	attributes property.Values,
+	serviceName string,
+	projectID object.ID,
+	environmentID object.ID,
+) error {
+	attrs := make(map[string]any, len(attributes))
+	for k, v := range attributes {
+		attrs[k] = string(json.ShouldMarshal(v))
+	}
+	opts := terraform.ServiceOpts{
```
Comment (thxCode) [inline pkg/apis/service/basic_view.go:356]: give a newline here

**Review by thxCode — APPROVED** [review]
LGTM

**Review by alexcodelf — APPROVED** [review]
LGTM

**Pull request merged**